packages/router/examples/simple.rs

```diff
@@ -37,10 +37,10 @@ fn BlogList(cx: Scope) -> Element {
     cx.render(rsx! { div { "Blog List" } })
 }
 
-fn BlogPost(cx: Scope) -> Element {
```

Comment:
Commenting this out causes the test to stop compiling because it is referenced in line 25